typings/index.d.ts

```diff
@@ -2340,6 +2345,7 @@ declare module 'discord.js' {
     restSweepInterval?: number;
     retryLimit?: number;
     presence?: PresenceData;
+    intents: BitFieldResolvable<IntentsString, number> | number;
```

Comment:
```suggestion
    intents: BitFieldResolvable<IntentsString, number>;
```
that's covered by https://github.com/discordjs/discord.js/blob/530ba35e681708bc1ad8489e7eb0b30885fdf02a/typings/index.d.ts#L2237 already